l3kernel/l3prg.dtx

```diff
@@ -91,12 +108,28 @@
 %   for existing definitions and perform assignments globally
 %   (\emph{cf.}~\cs{cs_new:Npn}) whereas the \texttt{set} versions do no
 %   check and perform assignments locally (\emph{cf.}~\cs{cs_set:Npn}).
-%   The conditionals created are dependent on the comma-separated list
-%   of \meta{conditions}, which should be one or more of \texttt{p},
-%   \texttt{T}, \texttt{F} and \texttt{TF}.
+%   The \Arg{conditions} argument is purely a reminder to the programmer:
+%   |T|, |F|, |TF| and |p| forms are always created.
+% \end{function}
+%
+% \begin{function}[added = 2024-11-22]
+%   {
+%     \prg_new_protected_conditional:Npn, \prg_set_protected_conditional:Npn, \prg_gset_protected_conditional:Npn,
+%     \prg_new_protected_conditional:Nn,  \prg_set_protected_conditional:Nn, \prg_gset_protected_conditional:Nn
+%   }
+%   \begin{syntax}
+%     \cs{prg_new_protected_conditional:Npn} \cs[no-index]{\meta{name}:\meta{arg spec}} \meta{parameters} \Arg{code} \\
+%     \cs{prg_new_protected_conditional:Nn} \cs[no-index]{\meta{name}:\meta{arg spec}} \Arg{code}
+%   \end{syntax}
+%   These functions create a family of protected conditionals using the
+%   same \Arg{code} to perform the test created. The \meta{code} does
+%   not need to be expandable.  The \texttt{new} version check for
+%   existing definitions and perform assignments globally
+%   (\emph{cf.}~\cs{cs_new:Npn}) whereas the \texttt{set} version do
+%   not (\emph{cf.}~\cs{cs_set:Npn}).
```

Comment:
```suggestion
%   not need to be expandable.  The \texttt{new} versions check for
%   existing definitions and perform assignments globally
%   (\emph{cf.}~\cs{cs_new:Npn}) whereas the \texttt{set} versions do
%   not (\emph{cf.}~\cs{cs_set:Npn}).
```

Copied from doc of `\prg_new_protected_conditional:Np(n)n` for which you may want to fix outside of the PR.

Reply:
> Copied from doc of `\prg_new_protected_conditional:Np(n)n` for which you may want to fix outside of the PR.

Sorted in 01cc6526 (Correct typos, 2024-11-24).